aries-site/src/pages/components/card.mdx

```diff
@@ -98,6 +109,80 @@ TBD
 - Cards that are less promotional should still have enticing titles but do not necessarily need as strong of visual emphasis. The white brand-approved textures are appropriate if a subtle visual emphasis is desired.
   - A use case is a software upgrade announcement.
 
+#### Examples
+
+These are example layouts for various use cases. Call-to-action ards that are highly promotional, such as event promotions, incorporate stronger visual elements while those that are less promotional, like software updates, are more subtle.
+
+<Example
+  componentName="Card"
```

Comment:
I believe nexus is written as NeXus or do we want to capitalize the "N" only?

Reply:
I copy and pasted directly from a design. I'm not familiar with NeXus vs Nexus, so would defer to someone more familiar.

Reply:
Truthfully, I googled it lol. I saw it written it both ways so I think either is fine.